packages/kotti-datepicker/src/DateInput.vue

```diff
@@ -54,6 +56,13 @@ export default {
 			if (this.currentDate) return this.currentDate.toLocaleDateString()
 			return null
 		},
+		hasLabel() {
+			return !(this.label === null || this.label === undefined)
+		},
+		labelRep() {
+			if (!this.hasLabel) return
```

Comment:
you should return an empty string instead of returning undefined. Otherwise, add a v-if condition to the span that check ```hadLabel``` instead of this.